backend/src/libs/interfaces/service.interface.ts

```diff
@@ -5,7 +5,7 @@ interface IService<T = unknown> {
   }>;
   create(payload: unknown): Promise<T>;
   update(id: unknown, payload: unknown): Promise<T>;
-  delete(): Promise<boolean>;
+  delete(id: unknown, payload: unknown): Promise<boolean>;
```

Comment:
1 - We may not need to delete comments
2 - Default delete should work from id
```suggestion
  delete(): Promise<boolean>;
```

Reply:
Can you clarify second point please ?

Reply:
Andrew've added to common interface args with payload on `delete` method -- common delete method should get only `id` for deletion in general

Reply:
Yes, I saw those changes and delete methods in comment repository and service. I just don't undestand what I need to do - rewrite using only id or remove entirely ?

Reply:
Just remove changes